alacritty_terminal/src/input.rs

```diff
@@ -455,7 +455,11 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
             if let Some(url) =
                 self.ctx.terminal().urls().drain(..).find(|url| url.contains(buffer_point))
             {
-                return MouseState::Url(url);
+                if self.ctx.selection_is_empty() &&
+                   self.ctx.mouse().left_button_state != ElementState::Pressed
```

Comment:
Based on my testing, this shouldn't be necessary. I've asked about this before, but could you go into more detail why this is needed?